splunk_add_on_ucc_framework/utils.py

```diff
@@ -34,7 +34,7 @@ def get_j2_env() -> jinja2.Environment:
     )
 
 
-def recursive_overwrite(src: str, dest: str) -> None:
+def recursive_overwrite(src: str, dest: str, ui_source_map: bool = False) -> None:
```

Comment:
nit: please add `ui_source_map` argument to "Args" section below.

Reply:
done